### ISSUE TITLE
[IMPROVEMENT] Tap app and server version to copy to clipboard

### ISSUE THREAD

**Issue by diegolmello** [RocketChat/Rocket.Chat.ReactNative#1415]
We should turn `Version` and `Server version` into touchable and copy the content to clipboard.
It's a nice way to get feedback.

<img src="https://user-images.githubusercontent.com/804994/69572639-53e7af80-0fa3-11ea-90aa-433f30732124.png" width="350" />

**Comment by Prateek93a** [comment]
@diegolmello Sir, here, do you prefer longpress or tap to copy the contents?

**Comment by diegolmello** [comment]
Hi, @Prateek93a.
A single tap is ok.

**Comment by Prateek93a** [comment]
@diegolmello Sir, I am facing the issue - Error: Unable to resolve module `react-native-keycommands`. I tried reinstalling the module, but that got stuck in the middle of installing. I tried to run by reseting cache but it didn't help.Kindly suggest something here.

**Comment by diegolmello** [comment]
It could be lots of reasons, but try this.
```
yarn
cd ios
rm -rf build
pod install
```

**Comment by Prateek93a** [comment]
@diegolmello Thanks for the solution Sir. Actually I am using android device so I could not implement the above solution. What worked for me was that I changed the import statement to-
import KeyCommands from 'react-native-keycommands/src';
I read that one possible reason of error could be that the program was unable to detect index.js file. So I did the above change. I do not know if its a good solution. Kindly provide your views. Thanks.
